baselines/dom.generated.d.ts

```diff
@@ -7112,7 +7112,7 @@ interface IDBRequestEventMap {
 }
 
 interface IDBRequest extends EventTarget {
-    readonly error: DOMError;
+    readonly error: DOMException | DOMError;
```

Comment:
Then we lose to know this difference through TS type. Type names are important info too.